docs/data/sql_functions.yml

```diff
@@ -687,7 +687,7 @@ collection:
     description: Returns a map created from an arrays of keys and values. Note that the lengths of two arrays should be the same.
   - sql: ARRAY_EXCEPT(array1, array2)
     table: arrayOne.arrayExcept(arrayTwo)
-    description: Returns an ARRAY that contains the elements from array1 that are not in array2. If no elements remain after excluding the elements in array2 from array1, the function returns an empty ARRAY. If one or both arguments are NULL, the function returns NULL. The order of the elements from array1 is kept.
+    description: Returns an ARRAY that contains the elements from array1 that are not in array2. If no elements remain after excluding the elements in array2 from array1, the function returns an empty ARRAY. If one or both arguments are NULL, the function returns NULL. The order of the elements from array1 is kept however the duplicates are removed.
```

Comment:
nit: what about following the format with 'ARRAY_UNION'?
```
Returns an ARRAY that contains the elements from array1 that are not in array2, without duplicates.
...
```

Reply:
yep, that looks more concise and easier to read

Reply:
Sorry, perhaps I haven't made myself clear, I want to keep the latter part the same. 
```
Returns an ARRAY that contains the elements from array1 that are not in array2, without duplicates. If no elements remain after excluding the elements in array2 from array1, the function returns an empty ARRAY. If one or both arguments are NULL, the function returns NULL. The order of the elements from array1 is kept.
```